src/Microsoft.AspNet.Mvc.Core/ModelBinding/BodyModelBinder.cs

```diff
@@ -64,9 +64,9 @@ public BodyModelBinder()
                     // system to skip other model binders and never to fall back.
                     return new ModelBindingResult(modelBindingKey);
                 }
-
-                var valueProviderResult = new ValueProviderResult(rawValue: model);
-                bindingContext.ModelState.SetModelValue(modelBindingKey, valueProviderResult);
+                
+                // We don't have anything useful to put in model state.
```

Comment:
> it's just something we did for consistency

We wanted to consistently make the bound values available in the `ModelStateDictionary`. Without either "overloading" `RawValue` or adding that new `ModelState` property, looks like a regression.

Note users can write into `ModelStateDictionary`. Also HTML generation converts _whatever_ is in `RawValue` to `string` and the few other types it needs.